ITMO_FS/filters/multivariate/FCBF.py

```diff
@@ -35,81 +34,35 @@ class FCBFDiscreteFilter(DataChecker):
     """
 
     def __init__(self):
-        self.selected_features = None
```

Comment:
there's no need to initialize it in init since we're going to overwrite it with an empty array in fit; there's also a check in estimator_checks that assures that no field is initialized except for what is passed as an init argument